docs/photoniq/faas/01-release-notes.md

```diff
@@ -3,5 +3,14 @@ sidebar_position: 2
 title: Release Notes
 ---
 
-## FaaS v1.001
-This represents the very first version of the PhotonIQ Function as a Service(FaaS)
\ No newline at end of file
+## May 2024 Release
+### v1.001
```

Comment:
We use semver notation for vesions: https://semver.org/
This needs to be 1.0.0

Reply:
fixed